portable/ARMv8M/non_secure/portable/GCC/ARM_CM23/portmacro.h

```diff
@@ -50,6 +50,7 @@
  */
 #define portARCH_NAME                    "Cortex-M23"
 #define portHAS_ARMV8M_MAIN_EXTENSION    0
+#define portHAS_ARMV8_1_M_EXTENSION      0
```

Comment:
A minor nit with the name `portHAS_ARMV8_1_M_EXTENSION`.  Architecture v8.1-M is not really an extension.  Extensions are "main", "MVE", "security", etc.  See A1.4 "Optional Extensions" in the [ARMv8-M Architecture Reference Manual](https://developer.arm.com/documentation/ddi0553/latest/).  ARMv8.1-M is an architecture like v8.0-M is.

Should we use `portARMV8M_MINOR_VERSION` instead?

Reply:
Hello @jefftenney,
Thanks for this good catch, since  ARMv8.1-M is not an extension, but addition of new features, some of which are optional like PACBTI and some of which are mandatory like PXN to the existing 8-M architecture. So I think `portSupports_ARMV8_1_M_ARCHITECTURE` would be reasonable, any thoughts?

Reply:
I think `portARMV8M_MINOR_VERSION` would work best. The minor version would be 0 for v8.0-M and 1 for v8.1-M. If there is ever a v8.2-M, the minor version would be 2.

(Please note I am not a member of the FreeRTOS team and not a maintainer. I am just a community contributor.)

Reply:
I like @jefftenney's suggestion. The check can then be - 

```c
#if ( portARMV8M_MINOR_VERSION  > 1 ) 
```

Reply:
@jefftenney Makes good sense, thanks for applying the suggestions @aggarg